syntax = "proto3";

service Xat {
	rpc SendMessage (MessageRequest) returns (stream MessageResponse);
}
message MessageRequest {
	string sender = 1;
	string receiver = 2;
	string text = 3;
}
message MessageResponse {
	string sender = 1;
	string text = 2;
}
